locksmith/src/fulfillment/dispatcher.ts

```diff
@@ -4,13 +4,22 @@ var HDWalletProvider = require('truffle-hdwallet-provider')
 const { WalletService, Web3Service } = UnlockJs
 
 export default class Dispatcher {
-  constructor(unlockAddress, purchasingAddress, credentials, host) {
+  purchasingAddress: string
+  unlockAddress: string
+  provider: any
```

Comment:
Instead of an `any`, can you copy what IntelliSense gives you as the type of `this.provider` on line 19?

Reply:
IntelliSense does return any for this :-(

Reply:
Fair, we can leave it